### ISSUE TITLE
conflict upgrading from 7.9 to 8.0 

### ISSUE THREAD

**Issue by groner** [takikawa/racket-ppa#10]
```
: kai@dipper kai; apt list --upgradable 
Listing... Done
racket-common/groovy,groovy 8.0+ppa1-5~groovy1 all [upgradable from: 7.9+ppa3-1~groovy1]
racket-doc/groovy,groovy 8.0+ppa1-5~groovy1 all [upgradable from: 7.9+ppa3-1~groovy1]
racket/groovy 8.0+ppa1-5~groovy1 amd64 [upgradable from: 7.9+ppa3-1~groovy1]
: kai@dipper kai; sudo apt upgrade
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Calculating upgrade... Done
The following packages will be upgraded:
  racket racket-common racket-doc
3 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Need to get 139 MB of archives.
After this operation, 113 MB of additional disk space will be used.
Do you want to continue? [Y/n] 
Get:1 http://ppa.launchpad.net/plt/racket/ubuntu groovy/main amd64 racket amd64 8.0+ppa1-5~groovy1 [90.8 MB]
Get:2 http://ppa.launchpad.net/plt/racket/ubuntu groovy/main amd64 racket-common all 8.0+ppa1-5~groovy1 [14.3 MB]
Get:3 http://ppa.launchpad.net/plt/racket/ubuntu groovy/main amd64 racket-doc all 8.0+ppa1-5~groovy1 [33.9 MB]
Fetched 139 MB in 6s (22.3 MB/s)                                                                                                                              
(Reading database ... 404124 files and directories currently installed.)
Preparing to unpack .../racket_8.0+ppa1-5~groovy1_amd64.deb ...
Unpacking racket (8.0+ppa1-5~groovy1) over (7.9+ppa3-1~groovy1) ...
dpkg: error processing archive /var/cache/apt/archives/racket_8.0+ppa1-5~groovy1_amd64.deb (--unpack):
 trying to overwrite '/usr/share/racket/collects/acks/compiled/acks_rkt.dep', which is also in package racket-common 7.9+ppa3-1~groovy1
dpkg-deb: error: paste subprocess was killed by signal (Broken pipe)
Preparing to unpack .../racket-common_8.0+ppa1-5~groovy1_all.deb ...
Unpacking racket-common (8.0+ppa1-5~groovy1) over (7.9+ppa3-1~groovy1) ...
Preparing to unpack .../racket-doc_8.0+ppa1-5~groovy1_all.deb ...
Unpacking racket-doc (8.0+ppa1-5~groovy1) over (7.9+ppa3-1~groovy1) ...
Errors were encountered while processing:
 /var/cache/apt/archives/racket_8.0+ppa1-5~groovy1_amd64.deb
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

`apt remove racket && apt install racket` worked to get around the issue.

**Comment by evdubs** [comment]
I also experienced this "paste subprocess was killed by signal" error on two machines. I fixed it by running:

`apt --fix-broken install`

**Comment by takikawa** [comment]
Thanks for the report. It looks like I should've set some package flags to make the package manager handle the upgrade properly. I'll look into fixing it.

**Comment by takikawa** [comment]
Should be fixed now in version `8.0+ppa1-6`
